Yoast/wordpress-seo
# Undeleted mistakenly removed tablesorter script, used in sitemaps

## Summary

This PR can be summarized in the following changelog entry:

Fixed table sort in sitemap interface.

## Relevant technical choices:

* undeleted script file, which was removed by mistake.

## Test instructions

This PR can be tested by following these steps:

* observe sort by table header click works in sitemaps and there are no JS errors.
